### PR TITLE
add OremangaNet.mjs

### PR DESCRIPTION
**Pull request by YainuWan11**
add file OremangaNet.mjs

### CODE REVIEW

File: src/web/mjs/connectors/OremangaNet.mjs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+export default class OremangaNet extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'oremanga';
+        super.label = 'Oremanga';
+        this.tags = [ 'manga', 'webtoon', 'thai' ];
+	this.url = 'https://www.oremanga.net';
+	this.path = '/manga-list'
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/OremangaNet.mjs]: tiny fix : its '/manga-list/', you miss a '/'  :p

File: src/web/mjs/connectors/OremangaNet.mjs [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+	this.queryMangas = 'div.mangalist-blc ul li a.series';
+	this.queryChapters = 'ul.series-chapterlist li div.flexch-infoz a';
+	this.queryPages = 'div.reader-area p source';
+	this.queryMangaTitle = 'div.series-title h2';
+	}
+async _getMangaFromURI(uri) {
```
Comment (MikeZeDev) [inline src/web/mjs/connectors/OremangaNet.mjs]: You may want to fix indentation , Linter will yell :)

**Review by MikeZeDev — CHANGES_REQUESTED** [review]
Nice. That said, we need an icon. Put a png file (64x64 px) without extension with name matching the connector id :  ``` src/web/img/connectors/oremanga ```

**Comment by MikeZeDev** [comment]
I cant get pictures at all. Again, was it tested? 
Also there is bloat in chapter name : the date.


I fixed all that, just add an icon

**Comment by YainuWan11** [comment]
Thank you,And I already added the icon.

**Comment by 27136773** [comment]
Should we really allow to add new websites to HakuNeko?
It will get out of sync with HaruNeko...

**Comment by MikeZeDev** [comment]
I think it would be reasonable to temporally freeze the addition of new websites in Hakuneko from contributors too.
I already stopped myself coding website for Hakuneko. I just fixes stuff for now.
By doing that we can concentrate on Haru.

On the other hand : 
* Its not like we have many submissions
* User submitted stuff is simple in general

We could make this one the last addition to Hakuneko for now?

**Comment by MikeZeDev** [comment]
> Thank you,And I already added the icon.

yes but, i told you to respect the ID naming scheme. ID are all lowercase. See al other icons name? :D

**Comment by 27136773** [comment]
[Missing in HaruNeko](https://github.com/search?q=repo%3Amanga-download%2Fharuneko%20%27oremanga%27&type=code)

**Comment by MikeZeDev** [comment]
No need to request more review if you dont change what is needed to change.

>yes but, i told you to respect the ID naming scheme. ID are all lowercase. See all other icons name? :D

Rename the icon and its ok.

**Comment by YainuWan11** [comment]
Already edited name the icon

**Review by MikeZeDev — APPROVED** [review]

**Pull request merged**